libraries/render-utils/src/AntialiasingEffect.h

```diff
@@ -216,10 +217,10 @@ class Antialiasing {
 };
 
 
-#else 
-class AntiAliasingConfig : public render::Job::Config {
+#else // User setting for antialias mode will probably be broken.
+class AntiAliasingConfig : public render::Job::Config { // Not to be confused with AntialiasingConfig...
     Q_OBJECT
-    Q_PROPERTY(bool enabled MEMBER enabled)
+    Q_PROPERTY(bool enabled MEMBER enabled) // Not sure if still useful.
```

Comment:
Probably is, this is likely called by VrMenu.cpp

Reply:
Ok, I'll remove the comment.